clients/python-wrapper/lakefs/object.py

```diff
@@ -560,10 +559,10 @@ def _upload_presign(self) -> lakefs_sdk.ObjectStats:
             headers["x-ms-blob-type"] = "BlockBlob"
 
         self._fd.seek(0)
-        resp = urllib3.request(method="PUT",
-                               url=url,
-                               body=self._fd,
-                               headers=headers)
+        resp = self._client.sdk_client.staging_api.api_client.rest_client.pool_manager.request(method="PUT",
```

Comment:
not in this pr - we really need to expose the client from our sdk client

Reply:
Agree, though currently we only have 2 usecase for it